### DISCUSSION TITLE
高分屏上无法扫描二维码

### DISCUSSION THREAD

**Discussion by sleikang** [messense/aliyundrive-webdav#568] [Q&A]
### 问题描述

![image](https://user-images.githubusercontent.com/23020770/185637281-dc25f21c-755b-48ab-86ae-da0b8b682c30.png)
高分屏上无法扫描二维码，建议输出png

### 重现步骤

4k显示器

### 版本

1.10.1

### 运行平台

Windows

### refresh token 获取方式

网页版

### 日志

高分屏上无法扫描二维码

**Comment by messense** [comment]
Windows 用 Powershell 了么

**Reply by sxul** [reply]
实测用 powershell 也这样，换个字体就好了

**Reply by sxul** [reply]
另外应该跟分辨率没有直接关联，我这里1080p也是这个输出
